MODULE_ACCEPTANCE_CRITERIA.MD

```diff
@@ -58,8 +58,13 @@ Please see [Before Development](MODULE_EVALUATION_TEMPLATE#before-development) f
   * _This is not applicable to libraries_
 * Must not depend on a FOLIO library that has not been approved through the TCR process
 * Gracefully handles the absence of third party systems or related configuration. (3, 5, 12)
-* Sonarqube hasn't identified any security issues, major code smells or excessive (>3%) duplication (6); and any disabled or intentionally ignored rules/recommendations are reasonably justified.
+* An automated code scannner hasn't identified any security issues, major code smells or excessive (>3%) duplication (6); and any disabled or intentionally ignored rules/recommendations are reasonably justified.
   * See [Rule Customization](https://dev.folio.org/guides/code-analysis/#rule-customization) details. 
+  * The code scanner may be Sonar (via Jenkins), github Actions or other. The testing algorithm must be set up, maintained and documented by the development team. The tests need to be reproducible by the Community.
```

Comment:
> testing algorithms

What is meant by `testing algorithm` in this context?

I'm concerned that these changes are mixing together automated testing and static code analysis

Personally, I consider those to be separate activities with different purposes

Reply:
> According to the wiki, you are both members of the sub-group. Is that correct?

Yes.

Reply:
I asked in the group chat if the other sub group members have a better understanding of what is the scope of this group. In particular, concerning changing the acceptance criteria of automated tests. If we should not find agreement there, I agree that this should be a topic of a Monday TC meeting.

Reply:
I think that Marc has a point, we use SonarQube as a tool to identify code smell, bugs and potential security issues. These are the domain of static code analysis. 

SonarQube  is also used to enforce a standard for code coverage as reported by the coverage tools that are employed for the specific language. This is not, strictly speaking, part of the static code analysis.

I do think this is valid, but it was a nuance that was not apparent to myself of the other members (at least as far as our discussions went.)

Given that, we should most likely revisit the scope in council to nail down specifically what this group is formed to address.